lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -117,7 +117,7 @@ def ask(message, options = {})
         # add space at the end as otherwise highline would add new line there :/
         message = "#{message} " unless message =~ /\s\Z/
         answer = @hl.ask(message) { |q| q.echo = false if options[:password] }
-        answer.to_s.chomp.downcase if answer
+        answer.to_s.chomp if answer
```

Comment:
The ask() method is used in multiple places now, and not downcasing may cause side-effects. [PR#126](https://github.com/theforeman/foreman_maintain/pull/126) implements a longer alternative for the same.